src/main/java/Deadline.java

```diff
@@ -0,0 +1,36 @@
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    protected LocalDate date;
+
+    protected LocalTime time = null;
+
+    public Deadline(String description, String by) throws EmptyDescriptionException {
+        super(description);
+        String[] str = by.split(" ");
```

Comment:
Variable name could be more specific. Maybe something like String[] tokens = by.split(" ")?